<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>admin_dialog</class>
 <widget class="QDialog" name="admin_dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1126</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTabWidget" name="Admin1">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1251</width>
     <height>691</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{
background-color : #181DA4 ;
border-radius : 20px ;
color : white ;
}</string>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <property name="iconSize">
    <size>
     <width>30</width>
     <height>30</height>
    </size>
   </property>
   <widget class="QWidget" name="tab_4">
    <attribute name="icon">
     <iconset resource="resources.qrc">
      <normaloff>:/new/prefix1/60.png</normaloff>:/new/prefix1/60.png</iconset>
    </attribute>
    <attribute name="title">
     <string>Home</string>
    </attribute>
    <widget class="QPushButton" name="pushButton_livre_dialog">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>200</y>
       <width>230</width>
       <height>80</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Livre Dialog</string>
     </property>
     <property name="icon">
      <iconset resource="resources.qrc">
       <normaloff>:/new/prefix1/10.png</normaloff>:/new/prefix1/10.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>60</width>
       <height>60</height>
      </size>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_adherents">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>420</y>
       <width>230</width>
       <height>80</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Adherent Dialog</string>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>49.png</normaloff>49.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>60</width>
       <height>60</height>
      </size>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_emprunts">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>310</y>
       <width>230</width>
       <height>80</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Emprunts_Dialog </string>
     </property>
     <property name="icon">
      <iconset resource="resources.qrc">
       <normaloff>:/new/prefix1/66.png</normaloff>:/new/prefix1/66.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>60</width>
       <height>60</height>
      </size>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>-20</x>
       <y>20</y>
       <width>511</width>
       <height>251</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>14.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>450</x>
       <y>80</y>
       <width>581</width>
       <height>491</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>6.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>990</x>
       <y>50</y>
       <width>41</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{Background-color: rgb(255, 255, 255);}</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>80.png</normaloff>80.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>40</width>
       <height>40</height>
      </size>
     </property>
    </widget>
    <zorder>label_6</zorder>
    <zorder>pushButton_livre_dialog</zorder>
    <zorder>pushButton_adherents</zorder>
    <zorder>pushButton_emprunts</zorder>
    <zorder>label_7</zorder>
    <zorder>pushButton</zorder>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="icon">
     <iconset>
      <normaloff>70.png</normaloff>70.png</iconset>
    </attribute>
    <attribute name="title">
     <string>Admin Access</string>
    </attribute>
    <widget class="QTableView" name="tableView_user">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>80</y>
       <width>331</width>
       <height>361</height>
      </rect>
     </property>
    </widget>
    <widget class="QTableView" name="tableView_admin">
     <property name="geometry">
      <rect>
       <x>750</x>
       <y>80</y>
       <width>321</width>
       <height>361</height>
      </rect>
     </property>
    </widget>
    <widget class="Line" name="line">
     <property name="geometry">
      <rect>
       <x>440</x>
       <y>50</y>
       <width>31</width>
       <height>461</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
    <widget class="QGroupBox" name="groupBox">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>40</y>
       <width>401</width>
       <height>511</height>
      </rect>
     </property>
     <property name="title">
      <string>Users </string>
     </property>
     <widget class="QPushButton" name="pushButton_delete_user">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>430</y>
        <width>90</width>
        <height>40</height>
       </rect>
      </property>
      <property name="text">
       <string>Delete</string>
      </property>
      <property name="icon">
       <iconset resource="resources.qrc">
        <normaloff>:/new/prefix1/45.png</normaloff>:/new/prefix1/45.png</iconset>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_load_user">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>430</y>
        <width>80</width>
        <height>40</height>
       </rect>
      </property>
      <property name="text">
       <string>Load</string>
      </property>
      <property name="icon">
       <iconset resource="resources.qrc">
        <normaloff>:/new/prefix1/42.png</normaloff>:/new/prefix1/42.png</iconset>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="geometry">
      <rect>
       <x>470</x>
       <y>40</y>
       <width>621</width>
       <height>501</height>
      </rect>
     </property>
     <property name="title">
      <string>Admins</string>
     </property>
     <widget class="QPushButton" name="pushButton_load_admin">
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>440</y>
        <width>80</width>
        <height>40</height>
       </rect>
      </property>
      <property name="text">
       <string>Load</string>
      </property>
      <property name="icon">
       <iconset resource="resources.qrc">
        <normaloff>:/new/prefix1/42.png</normaloff>:/new/prefix1/42.png</iconset>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_update_admin">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>310</y>
        <width>90</width>
        <height>40</height>
       </rect>
      </property>
      <property name="text">
       <string>Update</string>
      </property>
      <property name="icon">
       <iconset resource="resources.qrc">
        <normaloff>:/new/prefix1/59.png</normaloff>:/new/prefix1/59.png</iconset>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_delete_admin">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>310</y>
        <width>90</width>
        <height>40</height>
       </rect>
      </property>
      <property name="text">
       <string>Delete</string>
      </property>
      <property name="icon">
       <iconset resource="resources.qrc">
        <normaloff>:/new/prefix1/45.png</normaloff>:/new/prefix1/45.png</iconset>
      </property>
     </widget>
     <widget class="QGroupBox" name="groupBox_3">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>90</y>
        <width>241</width>
        <height>181</height>
       </rect>
      </property>
      <property name="title">
       <string>Add New Admin</string>
      </property>
      <widget class="QPushButton" name="pushButton_add_admin">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>120</y>
         <width>110</width>
         <height>40</height>
        </rect>
       </property>
       <property name="text">
        <string>Add Admin</string>
       </property>
       <property name="icon">
        <iconset resource="resources.qrc">
         <normaloff>:/new/prefix1/33.png</normaloff>:/new/prefix1/33.png</iconset>
       </property>
      </widget>
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>40</y>
         <width>235</width>
         <height>57</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Username</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEdit_new_usr_admin"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>Password</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEdit_new_pwd_admin"/>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QPushButton" name="pushButton_init_admin">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>370</y>
        <width>60</width>
        <height>40</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="resources.qrc">
        <normaloff>:/new/prefix1/46.png</normaloff>:/new/prefix1/46.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>25</width>
        <height>25</height>
       </size>
      </property>
     </widget>
     <zorder>groupBox_3</zorder>
     <zorder>pushButton_load_admin</zorder>
     <zorder>pushButton_update_admin</zorder>
     <zorder>pushButton_delete_admin</zorder>
     <zorder>pushButton_init_admin</zorder>
    </widget>
    <zorder>groupBox_2</zorder>
    <zorder>groupBox</zorder>
    <zorder>tableView_user</zorder>
    <zorder>tableView_admin</zorder>
    <zorder>line</zorder>
   </widget>
   <widget class="QWidget" name="tab">
    <attribute name="icon">
     <iconset resource="resources.qrc">
      <normaloff>:/new/prefix1/29.png</normaloff>:/new/prefix1/29.png</iconset>
    </attribute>
    <attribute name="title">
     <string>About</string>
    </attribute>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>-10</x>
       <y>10</y>
       <width>241</width>
       <height>131</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>14.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>30</y>
       <width>571</width>
       <height>501</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="resources.qrc">:/new/prefix1/65.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
